sdk/servicebus/service-bus/src/index.ts

```diff
@@ -10,7 +10,7 @@ export {
   RetryOptions,
   TokenCredential,
   TokenType,
-  WebSocketOptions
+  WebSocketOptions,
```

Comment:
Nit: Is this due to a local error or a setting that has been checked in? I worry that the next person updating this file would end up reverting this

Reply:
Hm...the prettier plugin and me never get along. I just re-ran `npm run format` and checked in again.